lib/python/qmk/keymap.py

```diff
@@ -18,6 +18,19 @@
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 __KEYMAP_GOES_HERE__
 };
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+__ENCODERS_GO_HERE__
+};
+"""
```

Comment:
Separate the encoder stuff, so it will not be used in keymaps without encoders.

```suggestion
ENCODER_SUPPORT = """void encoder_update_user(uint8_t index, bool clockwise) {
__ENCODERS_GO_HERE__
};
"""
```